hacspec-scrambledb/scrambledb/src/data_types.rs

```diff
@@ -10,20 +10,26 @@ use oprf::coprf::coprf_online::{BlindInput, BlindOutput};
 
 /// A type for finalized pseudonyms, i.e. those which have been hardened for
 /// storage by applying a PRP.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+#[cfg_attr(test, derive(Hash))]
```

Comment:
Do you want to keep this in only for tests or in general?

Reply:
We only need it for building `HashSet`s during testing, so I left it like this.